src/Compilation.zig

```diff
@@ -185,6 +185,9 @@ astgen_wait_group: WaitGroup = .{},
 /// TODO: Remove this when Stage2 becomes the default compiler as it will already have this information.
 export_symbol_names: std.ArrayListUnmanaged([]const u8) = .{},
 
+bundle: bool = false,
```

Comment:
Are these needed at this scope? I would think that having them in `InitOptions` should be enough.

Reply:
Removed.